File: level1/python/최대공약수와 최소공배수 12940.py
def solution(n, m):
    for divisor in range(1, min(n, m)+1): # 최대공약수 구하기
        if n % divisor == 0 and m % divisor == 0:
            gcd = divisor

    lcm = n * m // gcd # 최소공배수 구하기
        
    return [gcd, lcm]
print(solution(2, 5)) # 테스트 코드

File: level1/javascript/최대공약수와 최소공배수 12940.js
function solution(n, m) {
    var answer = [];
    let gcd = getGcd(n, m);
    return [gcd, n*m/gcd]; // [최대공약수, 최소공배수]
}
function getGcd(n, m){
    for(var divisor = Math.min(n, m); divisor > 0; divisor--){
        if(n % divisor == 0 && m % divisor == 0)
            return divisor;
    }
}